src/libopensc/card-cac.c

```diff
@@ -1499,7 +1499,7 @@ static struct sc_card_operations cac_ops;
 
 static struct sc_card_driver cac_drv = {
 	"Common Access Card (CAC)",
-	"cac",
+	"CAC",
```

Comment:
Sure. That was the point of my comment to make sure what is the expected case (lower one) and where else I should align the name. If the above list of occurrences is complete (`card-piv.c`, `ctx.c` and `pkcs15-syn.c`), then it will be fixed in the commit b1779838.